ros2service/ros2service/verb/call.py

```diff
@@ -34,9 +36,6 @@ def add_arguments(self, parser, cli_name):
             help="Name of the ROS service to call to (e.g. '/add_two_ints')")
         arg.completer = ServiceNameCompleter(
             include_hidden_services_key='include_hidden_services')
-        arg = parser.add_argument(
-            'service_type',
-            help="Type of the ROS service (e.g. 'std_srvs/Empty')")
         arg.completer = ServiceTypeCompleter(
```

Comment:
This line adds a completer for the `service_type` argument. With the removal of the argument it now overwrites the completer of the previous field.

Reply:
Aye, of course. Too hasty first time through.